cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,4840 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f08ac68",
+   "metadata": {},
+   "source": [
+    "# Itteratively Find Improved Consensus Labels for Multiannotator Data using Cleanlab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44a0555b",
+   "metadata": {},
+   "source": [
+    "This example shows how to improve consensus labels established from data labeled by multiple annotators by combining the CROWDLAB algorithm with itteratively retraining a model. For an introductory tutorial on finding consensus labels with the multiannotator library see [Find Best Consensus Labels for Multiannotator Data using Cleanlab](linke to hui wen's). \n",
+    "\n",
+    "As our multiannotator dataset, we use a subset of [cifar-10h](https://github.com/jcpeterson/cifar-10h) (Peterson et al., 2019) multiannotator labeling dataset which is a collection of 2751 annotators each labeling 200 examples for all 10,000 test images of the original [cifar10](https://www.cs.toronto.edu/~kriz/cifar.html).\n",
```

Comment:
[nit] CIFAR-10 and CIFAR-10H are usually capitalized in literature (I believe it's an acronym for something)